std_msgs/Header header
geometry_msgs/Quaternion orientation
float64[9] orientation_covariance

geometry_msgs/Vector3 angular_velocity
float64[9] angular_velocity_covariance

geometry_msgs/Vector3 linear_acceleration
float64[9] linear_acceleration_covariance

geometry_msgs/Vector3 magnetic_field
float64[9] magnetic_field_covariance